### ISSUE TITLE
Invalid scope error on Canvas Pages scopes 

### ISSUE THREAD

**Issue by escull75** [ucfopen/UDOIT#821]
In IU's Canvas instance, we were hitting invalid scope errors when enforcing scopes on the API client according to https://github.com/ucfopen/UDOIT/blob/main/INSTALL_CANVAS.md. I noticed that the scopes for Pages use :url_or_id, instead of :url, for their parameter. 

![image](https://user-images.githubusercontent.com/6439114/171682329-afccf6e2-5a00-4cad-aba0-5128daa6e310.png)

Changing these scopes in the CanvasLms.php file seemed to fix the issue in our instance. 

#820

**Comment by jonespm** [comment]
Additionally we noticed that when trying to edit the API key that had these scopes defined, we were unable and it kept giving a error when clicking Save. I'm not sure if this is related to the changed names or not but does seem suspicious. I can't edit this key to change the scopes. No other API keys seem to have this problem.

I had to create a new API key, update the code to what's on the PR #820 and then change the API key in the UDOIT institution table to get this all to work again.

I guess if you didn't have scope enforcement on this probably wouldn't have been a problem. 

<img width="513" alt="image" src="https://user-images.githubusercontent.com/27447/174669705-3485bf9c-a611-4c3b-9b32-9db7da6291e8.png">
